docs/reference/triggers/kafka.md

```diff
@@ -78,6 +78,11 @@ For more information on Nuclio function configuration, see the [function-configu
   <br/>
   **Type:** `[]string`
 
+- <a id="version"></a>**`version`** - The version of Kafka that Sarama will assume it is running against(by default 0.11.0).
+  Version string should be in the formats `0.11.0.3` for pre-1.0.0 versions and `1.0.0` for 1.0.0 and above.
```

Comment:
Minimal supported version is 0.11.0

Reply:
@TomerShor do you want to mention this or what do you mean?

Reply:
Yes, mention it please